packages/adminrouter/extra/src/includes/server/master.conf

```diff
@@ -439,15 +439,15 @@ location /acs/api/v1 {
 
 # Group: DC/OS UI Update Service
 # Description: DC/OS UI Update Service API
-location ~ ^/dcos-ui-update-service/api(?<url>.*)$ {
+location ~ ^/dcos-ui-update-service/api/(?<api_path>.*)$ {
     access_by_lua_block {
-        auth.access_dcos_ui_update_service_api_endpoint();
+        auth.access_dcos_ui_update_service_api_endpoint(ngx.var.api_path);
```

Comment:
Wouldn't we need to change the function signature of this method in `/packages/adminrouter/extra/src/lib/auth/open.lua` to take a path or am I misunderstanding what happens here?

Reply:
(I would expect a similar change like the one you did in the other PR)

Reply:
The parameter would not be used, but it does look like the other [functions pass the value](https://github.com/dcos/dcos/blob/master/packages/adminrouter/extra/src/lib/auth/open.lua#L162-L164) so we should be consistent with that pattern.